PROTO_tests/tests/operator_test.py

```diff
@@ -0,0 +1,774 @@
+                        continue
+                    # arkouda implements but not numpy
+                    results["arkouda_minus_numpy"].append((expression, str(akres), False))
+                    continue
+                try:
+                    akres = do_op(ltype, rtype, lscalar, rscalar, True, op)
+                except RuntimeError as e:
+                    if "not implemented" or "unrecognized type" in str(
+                        e
+                    ):  # numpy implements but not arkouda
+                        results["numpy_minus_arkouda"].append((expression, str(e), True))
+                    else:  # both implement, but arkouda errors
+                        results["both_implement"].append((expression, str(e), True, False, False))
+                    continue
+                # both numpy and arkouda execute without error
+                try:
+                    akrestype = akres.dtype
+                except Exception:
+                    warnings.warn(
+                        f"Cannot detect return dtype of ak result: {akres} (np result: {npres})"
+                    )
+                    results["both_implement"].append((expression, str(akres), False, True, False))
+                    continue
+
+                if akrestype != npres.dtype:
+                    restypes = f"{npres.dtype}(np) vs. {akrestype}(ak)"
+                    results["both_implement"].append((expression, restypes, False, True, False))
+                    continue
+                try:
+                    akasnp = akres.to_ndarray()
+                except Exception:
+                    warnings.warn(f"Could not convert to ndarray: {akres}")
+                    results["both_implement"].append((expression, str(akres), True, False, False))
+                    continue
+                if not np.allclose(akasnp, npres, equal_nan=True):
+                    res = f"np: {npres}\nak: {akasnp}"
+                    results["both_implement"].append((expression, res, False, False, True))
+                    continue
+                # Finally, both numpy and arkouda agree on result
+                results["both_implement"].append((expression, "", False, False, False))
+
+        print(f'# ops not implemented by numpy or arkouda: {len(results["neither_implement"])}')
+        if verbose:
+            for expression, err in results["neither_implement"]:
+                print(expression)
+        print(f'# ops implemented by numpy but not arkouda: {len(results["numpy_minus_arkouda"])}')
+        if verbose:
+            for expression, err, flag in results["numpy_minus_arkouda"]:
+                print(expression)
+        print(f'# ops implemented by arkouda but not numpy: {len(results["arkouda_minus_numpy"])}')
+        if verbose:
+            for expression, res, flag in results["arkouda_minus_numpy"]:
+                print(expression, " -> ", res)
```

Comment:
These could all be wrapped in a single `if verbose` it looks like

Reply:
There are 3 separate loops, all inside a separate `if verbose`. Why can we not have a single `if` and then the loops under?